doc/user/content/sql/drop-type.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "DROP TYPE"
+description: "`DROP TYPE` removes a user-defined data type."
+menu:
+  main:
+    parent: 'sql'
+---
+
+{{< version-added v0.7.1 />}}
+
+`DROP TYPE` removes a user-defined [custom data type](../create-type). It cannot be used on default data types.
+
+## Syntax
+
+{{< diagram "drop-type.svg" >}}
+
+Field | Use
+------|-----
+_data_type_name_ | The name of the type to remove.
+`CASCADE` | Automatically drop objects that depend on the type (such as table columns, functions, operators, or other types)
+`IF EXISTS`  |  Issue a notice, but not an error, if the type doesn't exist.
+`RESTRICT`  |  Refuse to drop the type if any objects depend on it. This is the default.
+
+
+## Examples
+
+These examples use the custom drop types from the [CREATE TYPE examples](../create-type#examples).
+
+### DROP TYPE (no dependencies)
```

Comment:
These samples really need the additional context of what is happening from a fresh state of the world. If you need help coming up with those examples, feel free to ping me or @JLDLaughlin.